test/index-static.test.js

```diff
@@ -1,12 +1,24 @@
 var should = require('should');
 
-require('node-jsx').install();
-
-var indexStatic = require('../lib/index-static.jsx');
+var indexStaticWatcher = require('../lib/index-static-watcher').create();
 
 describe('index-static', function() {
+  var indexStatic;
+
+  this.timeout(10000);
```

Comment:
(was reading http://mochajs.org/#test-specific-timeouts, but didn't quite get it)
is this to increase Mocha's default timeout in case our bundled js takes longer to build?